### DISCUSSION TITLE
Blocked and Blocking Notes

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [marcusolsson/obsidian-projects#794] [Help]
I've tried to add a field `blocked_by` where I use a link to point to another note.
I've created a board and a table to filter by this field but it appears as non-clickable text.

Any plan to add the blocked-by functionality or notes dependencies in general?

**Comment by tosascott** [comment]
@LeonardoGentile, as long as `blocked_by` is a singular link, enabling `richText` for that field will make the links clickable for the Table, Gallery and Board views. Lists of links only work in the Table view now as far as I can tell.

1. Open your Projects tab and select the project of interest.
2. Open the **Table** view.
3. In the `blocked_by` column, next to the label, click on the `...` button and select **Configure field**.
4. Toggle the **Enable rich text formatting** and save the configuration.
